le_dash/es.py

```diff
@@ -134,4 +134,9 @@ def episode_lookup(**kwargs):
     resp = q.execute()
     if resp.hits.total == 1:
         return Episode(resp.hits[0])
+    elif resp.hits.total > 1:
+        results = []
```

Comment:
using a list comprehension would make this:

    return [Episode(x) for x in resp.hits]